server/controllers/mentorsControllers.js

```diff
@@ -0,0 +1,14 @@
+import Mentors from '../model/mentors';
+
+class MentorsController {
+  static getAllMentors(req, res) {
+    return res.status(200).json({
+      status: 200,
+      data: {
```

Comment:
Remove `{ mentors: Mentors },`. You should have `data: Mentors`

Reply:
I fixed that